### PR TITLE
refactor: rename reconcileContext to reconcileInfo

### PR DESCRIPTION
**Pull request by xlgao-zju**
### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [x] E2E test
- [ ] No code
- [ ] Manual test (add steps below)

Side effects

- [ ] Breaking backward compatibility

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

File: controllers/common/desiredphase/controller.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ type Reconciler struct {
 }
 
 // Reconcile the common chaos
-func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+func (r *Reconciler) Reconcile(info context.Context, req ctrl.Request) (ctrl.Result, error) {
```
Comment (STRRL) [inline controllers/common/desiredphase/controller.go:46]: it seems this line of change is not required

File: controllers/common/desiredphase/controller.go [hunk 2]
```diff
@@ -56,39 +56,38 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
...
 	}
 
-	reconcilerCtx := &reconcileContext{
+	reconcilerInfo := &reconcileInfo{
 		obj:          obj,
 		Reconciler:   r,
 		shouldUpdate: false,
 	}
-	return reconcilerCtx.Reconcile(req)
+	return reconcilerInfo.Reconcile(req)
```
Comment (STRRL) [inline controllers/common/desiredphase/controller.go:64]: `reconcilerInfo` -> `reconcileInfo`

**Review by STRRL — COMMENTED** [review]
rest LGTM!

**Review by STRRL — APPROVED** [review]
LGTM! 

Thanks for your contribution!

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**